examples/tutorial_derive/03_02_option_mult.md

```diff
@@ -15,6 +15,9 @@ name: []
 $ 03_02_option_mult_derive --name bob
 name: ["bob"]
 
+$ 03_02_option_mult_derive --name bob --name john
+name: ["bob", "john"]
+
 $ 03_02_option_mult_derive --name=bob
```

Comment:
Could you drop the syntax variants?

Thats more important for `03_02_option_derive` but isn't needed here as much and can distract from the intent of this example.

Reply:
Are you suggesting to drop the following examples?
 - `$ 03_02_option_mult_derive --name=bob`
 - `$ 03_02_option_mult_derive -n bob`
 - `$ 03_02_option_mult_derive -n=bob`
 - `$ 03_02_option_mult_derive -nbob`

Reply:
Correct